unity/ProjectSettings/EditorBuildSettings.asset

```diff
@@ -9,18 +9,18 @@ EditorBuildSettings:
     path: 
     guid: 00000000000000000000000000000000
   - enabled: 0
-    path: Assets/Physics/Physics Scenes/FloorPlan21_physics.unity
+    path: Assets/Scenes/FloorPlan21_physics.unity
     guid: eb51d5fc565bb4e76902739ab241c0ea
   - enabled: 0
-    path: Assets/Physics/Physics Scenes/FloorPlan8_physics.unity
+    path: Assets/Scenes/FloorPlan8_physics.unity
```

Comment:
Any idea what these are? Why are there only a subset?

Reply:
These are the files actively in the "Scenes to Build" if building from the Editor. This subset will change based on what scenes are added or deleted from the editor build window